playwright-dependencies/tests/smoke.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+
+test('smokeTest', async ({ page }) => {
+  const containerPort = process.env.CONTAINER_PORT_EXPORTED;
+  const url = "localhost:" + containerPort;
+  
+  await page.goto(url);
+
+  await page.getByRole('button', { name: 'Sign In' }).click();
+  await page.getByLabel('Email Address').click();
+  await page.getByLabel('Email Address').press('Home');
```

Comment:
@lucasmiranda0, why we need this?

Reply:
To do the login.

Reply:
@lucasmiranda0, please remove this step if it's not mandatory